From 56e659a0312557e1a84a1af13af69265197b81be Mon Sep 17 00:00:00 2001
From: Jun Zhang <jun.zhang@arm.com>
Date: Fri, 18 Aug 2023 11:53:47 +0800
Subject: [PATCH] Change the base address for rss_comms-virtio

This address is changed to match the new secure memory
region for OP-TEE allocated in TF-A.

Upstream-Status: Pending
Signed-off-by: Jun Zhang <jun.zhang@arm.com>
---
 .../se-proxy/config/kronos-opteesp/default_se-proxy.dts.in      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deployments/se-proxy/config/kronos-opteesp/default_se-proxy.dts.in b/deployments/se-proxy/config/kronos-opteesp/default_se-proxy.dts.in
index 5f2afd1..6f6e776 100644
--- a/deployments/se-proxy/config/kronos-opteesp/default_se-proxy.dts.in
+++ b/deployments/se-proxy/config/kronos-opteesp/default_se-proxy.dts.in
@@ -35,7 +35,7 @@
 		};
 		rss_comms-virtio {
 			/* Armv8 A Foundation Platform values */
-			base-address = <0x00000000 0xA03F0000>; /* shared memory for the communication between TS and RSS */
+			base-address = <0x00000000 0xFFFFC000>; /* shared memory for the communication between TS and RSS */
 			pages-count = <16>;
 			attributes = <0x3>; /* read-write */
 		};
--
2.25.1

